Assignment/HW5_SQL/HW5_Starter_Code/app/models.py

```diff
@@ -1,13 +1,45 @@
 import sqlite3 as sql
 
-def insert_data():
-    # SQL statement to insert into database goes here
+def insert_customer(first_name,last_name,company,email,phone,street_address,city,state,country,zip_code):
+	# SQL statement to insert into database goes here
+	with sql.connect('app.db') as con:
+		cur = con.cursor()
+		cur.execute('PRAGMA foreign_keys = ON')
+		cur.execute('INSERT INTO customers (first_name,last_name,company,email,phone) VALUES (?,?,?,?,?)',(first_name,last_name,company,email,phone))
+		cur.execute('INSERT INTO addresses (street_address,city,state,country,zip_code) VALUES (?,?,?,?,?)',(street_address,city,state,country,zip_code))

```

Comment:
The customer id needs to be inserted into the Address table along with the attribute values. It does not automatically populate within the Address table.